frontend/src/game/Constants.ts

```diff
@@ -6,21 +6,23 @@ export const UNIT_SIZE = 48;
 export const HALF_UNIT_SIZE = UNIT_SIZE / 2;
 export const UNIT_SCALE = UNIT_SIZE / 16;
 
-
-export const GAME_CONFIG : Phaser.Types.Core.GameConfig = {
-    type: Phaser.AUTO,
-    width: 640,
-    height: 640,
-    pixelArt: true,
-    backgroundColor: '#181a1b',
-    physics: {
-        default: 'arcade',
-        arcade: {
-            //debug: true,
-        }
+export const GAME_CONFIG: Phaser.Types.Core.GameConfig = {
+  type: Phaser.AUTO,
+  width: 640,
+  height: 640,
+  pixelArt: true,
+  backgroundColor: '#1a1a1a', // Might need to figure out how to make this work with light/dark theming
+  // render: {
+  //   transparent: true,
+  // },
```

Comment:
were you trying to make the background transparent?

Reply:
Yeah I played with it a bit but figured transparent wasn't a good idea with the current UI